ideas/87-new-protocol.md

```diff
@@ -0,0 +1,71 @@
+## Preamble
+
+    Idea: <to be assigned>
+    Title: New Status App communication protocol 
+    Status: Draft
+    Created: 2018-03-08
+
+
+## Summary
```

Comment:
A lot of this body is technical and more description, can we create a tldr non technical summary in a few sentences that someone who isn't a Clojure/Go programmer can understand?